sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTest.java

```diff
@@ -51,109 +54,205 @@ private FormTrainingClient getFormTrainingClient(HttpClient httpClient,
-     * Verifies receipt data for a document using source as file url.
+     * Verifies receipt data for a document using source as input stream data.
      */
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
-    public void recognizeReceiptSourceUrl(HttpClient httpClient, FormRecognizerServiceVersion serviceVersion) {
+    public void recognizeReceiptData(HttpClient httpClient, FormRecognizerServiceVersion serviceVersion) {
         client = getFormRecognizerClient(httpClient, serviceVersion);
-        receiptSourceUrlRunner((sourceUrl) -> {
+        receiptDataRunner((data, dataLength) -> {
             SyncPoller<OperationResult, List<RecognizedReceipt>> syncPoller =
-                client.beginRecognizeReceiptsFromUrl(sourceUrl);
+                client.beginRecognizeReceipts(data, dataLength, FormContentType.IMAGE_JPEG);
             syncPoller.waitForCompletion();
             validateReceiptResultData(syncPoller.getFinalResult(), false);
         });
     }
 
     /**
-     * Verifies receipt data for a document using source as file url and include content when includeTextContent is
-     * true.
+     * Verifies an exception thrown for a document using null data value.
      */
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
-    public void recognizeReceiptSourceUrlTextDetails(HttpClient httpClient,
+    public void recognizeReceiptDataNullData(HttpClient httpClient,
         FormRecognizerServiceVersion serviceVersion) {
         client = getFormRecognizerClient(httpClient, serviceVersion);
-        receiptSourceUrlRunnerTextDetails((sourceUrl, includeTextContent) -> {
+        assertThrows(RuntimeException.class, () ->
+            client.beginRecognizeReceipts(null, RECEIPT_FILE_LENGTH, FormContentType.IMAGE_JPEG));
+    }
+
+    /**
+     * Verifies content type will be auto detected when using receipt API with input stream data overload.
+     */
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
+    public void recognizeReceiptDataWithContentTypeAutoDetection(HttpClient httpClient,
+        FormRecognizerServiceVersion serviceVersion) {
+        client = getFormRecognizerClient(httpClient, serviceVersion);
+        SyncPoller<OperationResult, List<RecognizedReceipt>> syncPoller =
+            client.beginRecognizeReceipts(getContentDetectionFileData(RECEIPT_LOCAL_URL), RECEIPT_FILE_LENGTH,
+                null);
+        syncPoller.waitForCompletion();
+        validateReceiptResultData(syncPoller.getFinalResult(), false);
+    }
+
+    /**
+     * Verifies receipt data for a document using source as as input stream data and text content when
+     * includeTextDetails is true.
+     */
+    // Turn off the tests as there is service regression on the media type.
```

Comment:
Can you uncomment this, I think I missed updating this when I updated the `FormRcognizerAsyncClientTests` file?